src/paymentservice/src/main/resources/logback-splunk.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+    <include resource="org/springframework/boot/logging/logback/defaults.xml"/>
+    ​
+    <!-- You can override this to have a custom pattern -->
```

Comment:
Is this a log pattern we want to keep? If not we should change it now and set a standard going forward. If we keep it we should nix that comment.

Reply:
I can't answer that question now, this is base on the the pattern I use in JRCC ...

Reply:
all good!